frontend/src/System/Status/MoreInfo/MoreInfo.js

```diff
@@ -23,24 +23,38 @@ class MoreInfo extends Component {
           </DescriptionListItemDescription>
 
           <DescriptionListItemTitle>
-            {translate('Discord')}
+            {translate('Wiki')}
           </DescriptionListItemTitle>
           <DescriptionListItemDescription>
-            <Link to="https://discord.gg/r5wJPt9">discord.gg/r5wJPt9</Link>
+            <Link to="https://wiki.servarr.com/Radarr">Wiki</Link>
           </DescriptionListItemDescription>
 
           <DescriptionListItemTitle>
-            {translate('Wiki')}
+            {translate('Reddit')}
+          </DescriptionListItemTitle>
+          <DescriptionListItemDescription>
+            <Link to="https://www.reddit.com/r/Radarr/">Radarr</Link>
+          </DescriptionListItemDescription>
+
+          <DescriptionListItemTitle>
+            {translate('Discord')}
           </DescriptionListItemTitle>
           <DescriptionListItemDescription>
-            <Link to="https://wiki.servarr.com/Radarr">wiki.servarr.com/Radarr</Link>
+            <Link to="https://discord.gg/r5wJPt9">Radarr</Link>
           </DescriptionListItemDescription>
 
           <DescriptionListItemTitle>
             {translate('Donations')}
           </DescriptionListItemTitle>
           <DescriptionListItemDescription>
-            <Link to="https://opencollective.com/radarr">opencollective.com/radarr</Link>
+            <Link to="https://opencollective.com/radarr">Donate to Radarr</Link>
```

Comment:
The title is translated, now that the link text is changed to not be an actual link it should be translated as well.